src/databricks/sqlalchemy/dialect/__init__.py

```diff
@@ -74,7 +74,7 @@ class DatabricksDialect(default.DefaultDialect):
     driver: str = "databricks-sql-python"
     default_schema_name: str = "default"
 
-    preparer = DatabricksIdentifierPreparer
+    preparer = DatabricksIdentifierPreparer  # type: ignore
```

Comment:
why do we need an ignore directive here?

Reply:
This is unrelated to retries. Just something caught by mypy that i needed to go away. Our next project is to completely overhaul the SQLAlchemy dialect. I'll address this then.